src/applications/edu-benefits/10203/tests/pages/programDetails.unit.spec.jsx

```diff
@@ -21,7 +21,7 @@ describe('Program Details', () => {
       />,
     );
     expect(form.find('input').length).to.equal(5);
-    expect(form.find('select').length).to.equal(1);
+    expect(form.find('select').length).to.equal(2);
```

Comment:
We expect to find two `select` elements now because previously we only had a select for the state, whereas now we'd have one for state and one for country, correct?